src/com/gfredericks/test/chuck/clojure_test.cljc

```diff
@@ -76,15 +77,25 @@
      (capture-reports* reports# (fn [] ~@body))
      @reports#))
 
+(defn times [options]
+  (cond (map? options)     (:num-tests options tc.clojure-test/*default-test-count*)
+        (integer? options) options))
+
+(defn other-options [options]
```

Comment:
Yeah, I tried to make them private at first and then realized they'd be part of the macroexpansion. But I'm happy to move them somewhere else if you'd like the file organized differently. I just placed them above `qc-and-report-exception` because they are used in that macro.